src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -1063,6 +1064,52 @@ def get_issue(self, issue_id):
       logs.error('Failed to retrieve issue.', issue_id=issue_id)
       return None
 
+  def get_description(self, issue_id):
+    """Gets the content of the description for the issue with the given ID"""
+    try:
+      comments = self._execute(
+          self.client.issues().comments().list(issueId=str(issue_id)))
+      logs.info('google_issue_tracker: get_description comments: %s' % comments)
+      return next(
+          filter(lambda c: c['commentNumber'] == 1, comments['issueComments']),
+          None)['comment']
+    except IssueTrackerError as e:
+      if isinstance(e, IssueTrackerNotFoundError):
+        return None
```

Comment:
I think you can catch this exception first and handle it instead of using isinstance.